answerUtils.mjs

```diff
@@ -7,6 +7,10 @@ const __dirname = path.resolve()
 const allMaterial = glob.sync(__dirname + '/material/**/*.toml')
```

Comment:
I recommend using `path.join` for combining path parts. More portable.

Reply:
changed